app/Jobs/ImportTurboVotePosts.php

```diff
@@ -43,7 +43,7 @@ public function __construct($filepath)
     {
         $this->filepath = $filepath;
 
-        $this->stats = $this->statsInit();
+        $this->stats = statsInit();
```

Comment:
Sorry I'm late to the game, but why was `$this->` removed here. Is this calling the method on the trait?

Reply:
@DFurnes ah shoot! this is outdated from when I was trying it as a helper, I'll push up a fix now!